src/nvim/os/input.c

```diff
@@ -289,8 +289,7 @@ size_t input_enqueue(String keys)
     unsigned new_size
       = trans_special(&ptr, (size_t)(end - ptr), (char *)buf, FSK_KEYCODE, true, NULL);
 
-    if (new_size) {
-      new_size = handle_mouse_event(&ptr, buf, new_size);
+    if (new_size && (new_size = handle_mouse_event(&ptr, buf, new_size))) {
```

Comment:
The `continue;` should still be executed if `handle_mouse_event` returns `0`, otherwise if a key notation follows `<MouseMove>` (doesn't happen in Nvim TUI, but still possible) it won't be interpreted.

Reply:
Oh crap. DONE.